SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3247,6 +3291,20 @@ int64 USpatialNetDriver::GetActorEntityId(const AActor& Actor) const
 	return PackageMap->GetEntityIdFromObject(&Actor);
 }
 
+FVector USpatialNetDriver::GetServerPosition() const
+{
+	if (LoadBalanceStrategy != nullptr)
+	{
+		if (const UGridBasedLBStrategy* GridBasedLBStrategy =
+				Cast<UGridBasedLBStrategy>(LoadBalanceStrategy->GetLBStrategyForVisualRendering()))
```

Comment:
HACKY HACKY THIS IS DEFINITELY FINE NOTHING TO SEE HERE

Reply:
don't need to cast? all strategies default to returning zero vector.